docs/package.json

```diff
@@ -30,6 +30,7 @@
     "react-dropdown": "^1.9.2",
     "react-feather": "^2.0.9",
     "react-github-btn": "^1.2.0",
+    "request": "^2.88.2",
```

Comment:
we dont need the request package here right?

Reply:
ok!
sorry my bad, will remove it!